index.css

```diff
@@ -0,0 +1,68 @@
+@import url("https://argon-css.netlify.app/components/index.css");
+
+/* header search bar */
+
+.search__bar__container {
+  width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.search__bar {
+  margin: 0 auto;
+  width: 100%;
+  height: 3rem;
+  padding: 0 1.25rem;
+  font-size: 1rem;
+  border-radius: 1rem;
+  border: none;
+  background-color: var(--regal-blue);
+  outline: none;
+  color: white;
+}
+.search__bar:focus {
+  border: 2px solid #e879f9;
+  transition: 0.2s ease;
+}
+.search__bar:focus::-webkit-input-placeholder {
+  transition: opacity 0.45s ease;
+  opacity: 0;
+}
+.search__bar:focus::-moz-placeholder {
+  transition: opacity 0.45s ease;
+  opacity: 0;
+}
+.search__bar:focus:-ms-placeholder {
+  transition: opacity 0.45s ease;
+  opacity: 0;
+}
+
+.search__icon {
+  position: relative;
+  left: -30px;
+}
+/* header search bar */
+
+.hero-heading {
+  background: #e879f9;
+  background: -webkit-radial-gradient(
+    ellipse farthest-corner at center center,
+    #e879f9 0%,
+    #701a75 100%
+  );
+  background: -moz-radial-gradient(
+    ellipse farthest-corner at center center,
+    #e879f9 0%,
+    #701a75 100%
+  );
+  background: radial-gradient(
+    ellipse farthest-corner at center center,
+    #e879f9 0%,
+    #701a75 100%
```

Comment:
here too...use variables for colors

Reply:
also implement changes by previous reviewers

Reply:
Okay will make the changes.